### DISCUSSION TITLE
Aviary 0.9.6 preview

### DISCUSSION THREAD

**Discussion by jkirk5** [OpenMDAO/Aviary#585] [Announcements]
The next tagged version of Aviary will be arriving soon! There will be a number of small, backwards-incompatible changes to be aware of. Once v0.9.6 arrives, you may notice minor changes in your model's results. Here is a list of changes, and instructions on modifying your models to replicate old results.

**Key new features since last version (0.9.4-dev):**
- Off-design missions
	- Run your aircraft through alternate mission scenarios (after sizing and/or optimization)
	- Save your sized aircraft results to load and run off-design missions at a later time
- Electrified aircraft
	- A simple motor model (located in `Propulsion`) and simple battery model (located in `Energy`) are now available for use
	- These models currently must be added as external subsystems and require the use of level2 or deeper
- Propeller aircraft
	- Propeller modeling is now supported, by default using the Hamilton Standard methodology
	- A user-provided propeller performance map can be used as well
	- The `TurbopropModel` (a `EngineModel` subclass) packages together a shaft-power generating model, a gearbox model, and a propeller model. By default, a `TurbopropModel` attempts to build a turboshaft engine deck connected to a simple gearbox and the Hamilton Standard propeller module. Any of these models can be substituted with user-provided subsystems. The `TurbopropModel` currently requires use of the level2 interface or deeper.
- Non-homogenous engine sets
	- The use of multiple unique types of `EngineModels` on an aircraft is supported for energy method missions
	- Use of non-homogenous engines currently requires the level2 interface or deeper
- GUI improvements
	- The draw_mission GUI has been revamped to make mission generation easier. Mission profiles can be defined tabularly, and more phase info options have been added to the builder. Users can also change plot axes, and units for mission definition.
	- The new plot_drag_polar GUI can import a drag polar in Aviary .csv format for visualization
- Hundreds of bugfixes, minor feature additions, and other improvements!

**Non-backwards compatible changes:**
- The minimum OpenMDAO version supported by Aviary will be 3.35. All OpenMDAO outputs and reports are now sent to the "<your_model_name>_out" directory, and the Aviary dashboard will look there to populate tabs. The dashboard will not find files for models run using older versions of OpenMDAO unless they are manually moved to the expected place
- The maximum Numpy version supported by Aviary will be <2.0
- The `Atmosphere` component now defaults to using geodetic altitude, rather than geopotential. This may cause minor changes in results from components that use `Atmosphere`, such as mission analysis. To revert, reset the default on `aviary/subsystems/atmosphere/atmosphere.py` to `default='geodetic',`
- The default normalization scheme for EngineDecks has been changed to no longer use global throttle range (which can result in extrapolation of engine data). This can significantly change model results depending on what data is available in the engine data file. To replicate previous results, set `aircraft:engine:global_throttle` and `aircraft:engine:global_hybrid_throttle` to `True`

**Comment by stefano2734** [comment]
It is available since December 16.
